pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -256,6 +262,14 @@ func (r *ReconcileCephBlockPoolRadosNamespace) reconcile(request reconcile.Reque
 	}
 
 	r.updateStatus(r.client, namespacedName, cephv1.ConditionReady)
+
+	if csi.EnableCSIOperator() {
+		err = csi.CreateUpdateRadosNamespace(r.clusterInfo.Context, r.client, r.clusterInfo, cephBlockPoolRadosNamespacedName, buildClusterID(cephBlockPoolRadosNamespace), cephCluster.Name)
```

Comment:
This name is a bit misleading IMO, we are not creating `RadosNamespace` here, but creating the `clientProfile` CR that has reference to `radosNamespace` name.